<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assets</class>
 <widget class="QMainWindow" name="Assets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Triangulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_GL" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="Parameters_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>141</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string> Statistics</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertex</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>50</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Triangles</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="wireframe">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>411</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Queen settings</string>
       </property>
       <widget class="QRadioButton" name="radioShadingButton_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal (from cross)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="normal2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal (from laplacian)</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="curvature">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Curvature</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="heat">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>300</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Heat simulation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="heatReset">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>210</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset Heat</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="heatSimulate">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>210</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Simulate +0.1s</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="example">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>371</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load queen</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="bruh">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>411</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>Triangulation Settings</string>
       </property>
       <widget class="QLineEdit" name="tri_path">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>surfaces/alpes_random_1.txt</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QCheckBox" name="tri_elevate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>300</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Elevate surface</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>66</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_load">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>381</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load triangulation</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="tri_load_first">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>300</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Load only the first triangle</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_next">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>160</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add next point</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="tri_delaunay">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>300</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Delaunay</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="tri_load_delaunay">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>381</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load and convert from naive to delaunay</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="resetcameraButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset Camera</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionExport_obj">
   <property name="text">
    <string>OBJ File (.obj)</string>
   </property>
  </action>
  <action name="actionExport_ma">
   <property name="text">
    <string>Maya File (.ma)</string>
   </property>
  </action>
  <action name="actionLoad_ma">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionGL">
   <property name="text">
    <string>GL</string>
   </property>
  </action>
  <action name="actionVectorized">
   <property name="text">
    <string>Vectorized</string>
   </property>
  </action>
  <action name="actionExport_svg">
   <property name="text">
    <string>SVG File (.svg)</string>
   </property>
  </action>
  <action name="actionSphere_Tracing_png">
   <property name="text">
    <string>Sphere Tracing (.png)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
